### DISCUSSION TITLE
Device icon/type?

### DISCUSSION THREAD

**Discussion by alexeyvolkoff** [christgau/wsdd#176] [Q&A]
Hi!
How can one specify the device type showing up in Network to be not a "Computer", but "NAS"?

Thanks!

**Comment by christgau** [comment]
Wsdd does not show any icon. It only reports the device type. Currently, you cannot specify the reported device type. It is burned in https://github.com/christgau/wsdd/blob/2b1a7643fe7acd8229b8232ebf648c2bf882ff3c/src/wsdd.py#L368. One could think of adding an option for this. Do you know which device type needs to be reported for the device being reported as "NAS"? Btw: Where do you want to see the icon? Windows' Explorer?!

**Reply by alexeyvolkoff** [reply]
Yes, in Windows Explorer.  
I would like to show my device same way as WD MyCloud does, in "Storage" section 
![image](https://user-images.githubusercontent.com/5536986/235462411-fccf3d74-ee9d-45c0-9ab7-99150cbe0c94.png)

Unfortunately, I have no idea which type name matches this icon.

`
![image](https://user-images.githubusercontent.com/5536986/235462647-f6af79f5-b2db-40a9-83d3-170bdcbfab67.png)
`
